spec.md

```diff
@@ -447,8 +449,9 @@ The following rules apply if `parameters` is included anywhere in the catalog:
 [JSON Schema draft v4](http://json-schema.org/).
 * Platforms SHOULD be prepared to support later versions of JSON schema.
 * The `$schema` key MUST be present in the schema declaring the version of JSON
-schema being used.
-* Schemas MUST NOT contain any external references.
+schema being used unless the schema is composed of only a `$ref`.
+* Schemas MUST NOT contain any external references except for schemas found
+  under `/schemas{/*}` (see [$ref Usage](#ref-usage) for further information).
```

Comment:
is the correct path under which $refs can point: `/v2/catalog/schemas/...`  ?

the construct `{/*}` doesn't appear defined anywhere

Reply:
Tried to clear the wording up like this:

* Schemas MUST NOT contain any external references except for schemas found
  under `/v2/catalog/schemas` or `osb_v2:` style `$ref`s via
  `/v2/catalog/schemas/{:component_path}` (see [$ref Usage](#ref-usage) for
  further information).

My intent is to say `osb_v2` style external refs are ok, and standard `http://my.host.com/my/schema/doc.json#moreInfo` will work too as long as that can be found in `/v2/catalog/schemas`. It is subtle in the spec, and only people who understand JSON schema spec will understand this is allowed. But if you don't understand json schema spec you will get it wrong and break client side processing so I think this requirement is ok.